tests/playwright/src/runner/podman-desktop-runner.ts

```diff
@@ -192,17 +194,30 @@ export class PodmanDesktopRunner {
     }
 
     if (this.getElectronApp()) {
-      const elapsed = await this.trackTime(async () => await this.getElectronApp().close());
-      console.log(`Elapsed time of closing the electron app: ${elapsed} ms`);
+      const timeout = 30000;
+      console.log(`Closing Podman Desktop with a timeout of ${timeout} ms`);
+      try {
+        await Promise.race([
+          waitWhile(() => this.runningState(), { timeout: timeout, diff: 100 }),
```

Comment:
Can't we just use `async () => this.isRunning()` instead of creating duplicated function with promise return value?

Reply:
no, since that method is sync not async